sequana/scripts/main.py

```diff
@@ -351,25 +389,30 @@ def rnadiff(**kwargs):
     from sequana.featurecounts import FeatureCount
     from sequana.rnadiff import RNADiffAnalysis, RNADesign
     from sequana.modules_report.rnadiff import RNAdiffModule
+    from sequana import GFF3
 
     logger.setLevel(kwargs['logger'])
 
     outdir = kwargs['output_directory']
     feature = kwargs['feature_name']
     attribute = kwargs['attribute_name']
     design = kwargs['design']
-    reference=kwargs['reference']
+    reference = kwargs['reference']
+
+    if os.path.exists(outdir) and kwargs['force'] is False:
+        logger.error(f"{outdir} exist already. Use --force to overwrite")
+        sys.exit(0)
```

Comment:
Le `is False` j'ai du mal, je suis plus team `and not kwargs['force']`
Sinon vu que c'est une erreur c'est pas bon le `sys.exit(0)`